yapapi/executor/__init__.py

```diff
@@ -394,12 +420,10 @@ async def find_offers() -> None:
                             score,
                         )
                     except InvalidPropertiesError as err:
-                        emit(events.ProposalRejected(prop_id=proposal.id, reason=str(err)))
+                        await reject_proposal(proposal, "Malformed offer")
                         continue
                     if score < SCORE_NEUTRAL:
-                        with contextlib.suppress(Exception):
-                            await proposal.reject(reason="Score too low")
-                        emit(events.ProposalRejected(prop_id=proposal.id, reason="Score too low"))
+                        await reject_proposal(proposal, "Score too low")
```

Comment:
This is not required, because there is no code after all `if` branches, but maybe we can add `continue` after the call to show our intent? Feel free to ignore this suggestion.
```suggestion
                        await reject_proposal(proposal, "Score too low")
                        continue
```